src/composables/sdk13.ts

```diff
@@ -148,6 +150,31 @@ export function useSdk13({ store }: IDefaultComposableOptions) {
     return sdk;
   }
 
+  /**
+   * drySdk is the sdk instance with no accounts attached.
+   * To use for multisig operations.
+   */
+  async function getDrySdk(): Promise<AeSdk> {
+    if (!drySdk) {
+      const nodeInstance = await createNodeInstance(activeNetwork.value.url);
+      drySdk = new AeSdk({
+        nodes: [{
+          name: activeNetwork.value.name,
+          instance: nodeInstance!,
+        }],
+        onCompiler: new CompilerHttp(activeNetwork.value.compilerUrl),
```

Comment:
None of the dry SDK methods are calling the node or compiler. We should find a way to initialize drySDK without node and compiler

Reply:
For the one where we are sending the confirmed tx, we can use normal SDK and not dry sdk.